### DISCUSSION TITLE
archivo heads.h

### DISCUSSION THREAD

**Discussion by adribastiasm** [IIC2133-PUC/2024-1#80] [Tarea 1] [answered]
Holaa! Para la parte 2 de la tarea, el archivo heads.h lo puedo modificar para poder hacer los arboles de la forma que se me ocurre? o no se puede cambiar nada de ese archivo?
Gracias de antemano.
Saludos!

**Comment by sachondo7** [comment, marked as answer]
si puedes cambiarlo, queda a criterio tuyo cómo crear los structs. Lo importante es que cumplas con lo pedido en el enunciado. El código base es solo para guiarlos.

**Reply by adribastiasm** [reply]
perfecto, graciass
